dangerfile.js

```diff
@@ -0,0 +1,31 @@
+// eslint-disable-next-line import/no-extraneous-dependencies, no-unused-vars
```

Comment:
@bjankord I added the Danger Token to the travis-ci build. Is there any more setup for Danger thats needed?